inspect/api.go

```diff
@@ -53,10 +53,16 @@ func getRpcResponse(ctx context.Context, rpcClient *rpc.Client, respData interfa
 func GetNetworkID(ctx context.Context, rpcClient *rpc.Client) (*big.Int, error) {
 	var result string
```

Comment:
Can we change this to `resultStr` and then `resultHex` to `result` instead? The second one looks like we are trying a general fallback by leveraging `(*big.Int).MarshalText`.